### ISSUE TITLE
Check and Update various UI Elements to Comply with the Web Accessibility Protocol

### ISSUE THREAD

**Issue by mandywong0** [cs169/berkeley-reentry-student-program#59]
Use the [axe-core](https://www.fastruby.io/blog/testing-for-accessibility.html) ruby gem (axe-core-cucumber) for cucumber tests (automated check & write cucumber tests (expect axe-clean for each html page )) to check for current UI accessibility issues existing on each page of our current site. 
Based on the test results, update UI elements accordingly. 

If desired, can use tools such as the [WAVE tool](https://wave.webaim.org/) to perform manual, visual checks

Notes:
- look for failures that are global 
- use one feature file accessibility.feature and make a scenario for each page
